python/unityagents/environment.py

```diff
@@ -43,23 +43,23 @@ def __init__(self, file_name, worker_id=0,
         atexit.register(self._close)
         self.port = base_port + worker_id
         self._buffer_size = 12000
-        self._version_ = "API-3"
+        self._version_ = "API-4"
```

Comment:
Ensure that incompatibilities between old socket and RPC allow for graceful errors.

Reply:
Error is now graceful : 
```
unityagents.exception.UnityTimeOutException: The Unity environment took too long to respond. Make sure that :
	 The environment does not need user interaction to launch
	 The Academy and the External Brain(s) are attached to objects in the Scene
	 The environment and the Python interface have compatible versions.
```